Modules/_zstd/_zstdmodule.h

```diff
@@ -149,12 +47,6 @@ typedef enum {
     DICT_TYPE_PREFIX = 2
 } dictionary_type;
```

Comment:
Perhaps this should move to zstddict.h?

Reply:
I'd support moving it, but `DICT_TYPE_DIGESTED` is used by all three C files, so there's no easy way. My main goal here was to remove the `#include <zstd.h>` from the file, though, which is done.

Reply:
Okay, we can always change this in the future.